src/turnkeyml/build/export.py

```diff
@@ -571,9 +563,8 @@ def fire(self, state: build.State):
         inputs_file = state.original_inputs_file
         if os.path.isfile(inputs_file):
             inputs = np.load(inputs_file, allow_pickle=True)
-            to_downcast = False if state.quantization_samples else True
             inputs_converted = tensor_helpers.save_inputs(
-                inputs, inputs_file, downcast=to_downcast
+                inputs, inputs_file, downcast=True
```

Comment:
Is downcast as part of `save_inputs` only ever used for quantization sampels?
If so, can we get rid of that arg and also a few lines of code inside `save_inputs`?

Reply:
No, downcast=False is used in 3 places in this file